python/cudf/cudf/_lib/orc.pyx

```diff
@@ -159,6 +159,8 @@ cdef compression_type _get_comp_type(object compression):
         return compression_type.NONE
     elif compression == "snappy":
         return compression_type.SNAPPY
+    elif compression == "ZLIB":
```

Comment:
Do we also have to add `LZ4`, `LZO` & etc.. to this if/elif handling too?

Reply:
not at this time. Only these types are supported ATM